scene/gui/spin_box.cpp

```diff
@@ -51,7 +51,10 @@ void SpinBox::_text_entered(const String& p_string) {
 
 	//if (!p_string.is_numeric())
 	//	return;
-	set_val( p_string.to_double() );
+	String value = p_string;
```

Comment:
Why a new String? If it's for the readability of the name I would make it `const String&`.